dhall-json/src/Dhall/JSON.hs

```diff
@@ -153,7 +153,7 @@
 >       }
 >     ]
 
-> $ dhall-to-json <<< './example.dhall'
+> $ dhall-to-json --preserveNull <<< './example.dhall'
```

Comment:
I think it would be better to document the default behavior first. But please add a sentence to mention `--preserveNull`.